sdlib/d/gc/arena.d

```diff
@@ -116,8 +116,22 @@ public:
 		// TODO: in contracts
 		assert(isSmallSize(size));
 
+		void*[1] buffer;
+		auto gotSlots = allocSmallBatch(emap, size, buffer[0 .. 1]);
+		if (gotSlots == 0) {
+			return null;
+		}
+
+		return buffer[0];
+	}
+
+	uint allocSmallBatch(shared(ExtentMap)* emap, size_t size,
```

Comment:
Is the batch small? The good news is that it doesn't matter much: this is not claled, you can just tweak allocSmall and be done with it.